app/controllers/works_controller.rb

```diff
@@ -5,11 +5,11 @@ class WorksController < ApplicationController
   # only registered users and NOT admin should be able to create new works
   before_filter :load_collection
   before_filter :load_owner, :only => [ :index ]
-  before_filter :users_only, :except => [ :index, :show, :navigate, :search, :collected, :edit_tags, :update_tags ]
-  before_filter :check_user_status, :except => [ :index, :show, :navigate, :search, :collected ]
+  before_filter :users_only, :except => [ :index, :feed, :show, :navigate, :search, :collected, :edit_tags, :update_tags ]
```

Comment:
This was like this when I found it, so I'm not going to change it unless I have to.

The rest (and the test failure), I'll fix.